css/server.css

```diff
@@ -0,0 +1,12 @@
+#security-password {
```

Comment:
server.css isn't a very appropriate name. Often can make a difference when debugging (don't know nextcloud specifics)

Reply:
Actually, `server.css` is the name NC expects to find theme-level CSS in. This app not being a theme, it probably doesn't need to adhere to the convention.